docs/source/training_guide.rst

```diff
@@ -134,9 +210,34 @@ For the AutoML we just need the data and the configuration parameters. You can c
         wait_complete=True
     )
 
+See an example of the a configuration file:
+
+.. code-block:: json
+
+    {
+        "train_data": {
+            "file_type": "csv",
+            "sep": ",",
+            "file_name": "dados.csv"
+        },
+        "model_flow": "classification",
+        "target": "target",
+        "iterations": 1,
+        "metric": "ks",
+        "split_type": "random",
+        "stages": {
+            "models": ["logreg"]
+            }
+    }
+
+
+Running an AutoML training execution
```

Comment:
Saving a local training to Neomaril

Reply:
https://github.com/datarisk-io/mlops_codex/pull/68/commits/5d0442b7c9b8fe702477941f5e716c70e8ccf644